website/content/docs/connect/config-entries/partition-exports.mdx

```diff
@@ -0,0 +1,212 @@
+---
+layout: docs
+page_title: 'Configuration Entry Kind: Partition Exports'
+description: >-
+  The partition-exports configuration entry enables you to export services from a single file.
+  Settings in this configuration entry can apply to services in any namespace of the specified partition. Write access to the mesh resource is required.
+---
+
+# Partition Exports
+
+This topic describes the `partition-exports` configuration entry type. The `partition-exports` configuration entry enables Consul to export service instances to other admin partitions from a single file. This enables your services to be networked across admin partitions. See [Admin Partitions](/docs/enterprise/admin-partitions) for additional information.     
+
+-> **v1.11.0+:** This config entry is supported in Consul versions 1.11.0+.
+
+## Introduction
+
+You can configure Consul to export services contained in an admin partition to one or more additional partitions by declaring the `partition-exports` configuration entry in the `kind` field. This enables you to route traffic between services in different clusters that share a single set of Consul servers. 
+
+You can configure the settings defined in the `partition-exports` configuration entry to apply to all namespaces and federated datacenters.
+
+## Requirements
+
+* A Consul Enterprise binary
+* A partition that corresponds to the configuration entry. As in, the partition exports config entry for partition "frontend" requires that the "frontend" partition exists
+
+
+## Usage
+
+1. Verify that your datacenter meets the conditions specified in the [Requirements](#requirements).
+1. Specify the `partition-exports` configuration in the agent configuration file (see [`config_entries`](/docs/agent/options#config_entries)) as described in [Configuration](#configuration).
```

Comment:
This comment is still relevant: https://github.com/hashicorp/consul/pull/11558#discussion_r749708024

I don't think applying this config entry via agent configuration will be common. For starters only Consul servers can apply config entries via their agent configuration file. This means that for existing installations, Consul servers would need to be restarted to apply the config entry, which I doubt anyone will do.

If people are trying this out manually they're likely to specify an HCL file with the config entry, and apply it with the `consul config write` CLI command. Or if they're on Kubernetes (likely), they'll use CRDs: https://www.consul.io/docs/k8s/crds 

Speaking of which, that CRDs docs page should be updated to include a link to this new config entry. In the list of `We currently support the follow configuration entry kinds`